internal/ui/assets/css/styles.css

```diff
@@ -122,9 +122,37 @@ header.cdc-header .header-search {
   padding-top: 6px;
 }
 
-.vh {
-  height: 50vh;
-  background-color: orange;
+.usa-accordion__button {
+  margin-bottom: 12px;
+}
+
+.usa-button {
+  height: 3rem;
+}
+
+.usa-input {
```

Comment:
if you make this `max-width: 100%` and adjust the button as noted above, this should stretch everything to cover the full width of the container